src/tests/multihost/alltests/test_config_merging.py

```diff
@@ -115,6 +118,7 @@ def test_0004_formatsnippetfile(self, multihost):
         multihost.client[0].put_file_contents(snippet_file, file_content)
         cmd_chmod = 'chmod 600 %s' % snippet_file
         multihost.client[0].run_command(cmd_chmod, raiseonerr=False)
+        multihost.client[0].run_command(f"chmod o-rwx {snippet_file}")
```

Comment:
This line doesn't make sense because line above already sets chmod 600.

Reply:
By the way, what are the user:group of a file created by `put_file_contents()`?

Reply:
root:root